R/create_lkups.R

```diff
@@ -106,6 +106,39 @@ create_lkups <- function(data_dir, versions) {
     data_dir, ref_lkup$cache_id
   )
 
+  # Add data interpolation ID (unique combination of survey files used for one
+  # or more reporting years)
+  ref_lkup <- ref_lkup[, data_interpolation_id := paste(cache_id, pop_data_level, sep = "_")]
+  ref_lkup <- ref_lkup[, data_interpolation_id := paste(unique(data_interpolation_id), collapse = "|"),
+                       by = .(interpolation_id)]
+
+  # Create interpolation list.
+  # This is to facilitate interpolation computations
+  unique_survey_files <- unique(ref_lkup$data_interpolation_id)
+  interpolation_list <- vector(mode = "list", length = length(unique_survey_files))
+
+  for (i in seq_along(interpolation_list)) {
```

Comment:
This is minimal but `seq_len(length(interpolation_list))`  might be faster than `seq_along(interpolation_list)` But than again, seq_along might be more concise.

Reply:
Did some benchmarking, and did not find it was faster... Very small difference anyway